R/processCapabilityStudies.R

```diff
@@ -246,7 +245,7 @@ processCapabilityStudies <- function(jaspResults, dataset, options) {
     sdw <- qccFit[["std.dev"]]
     valueVector <- c(lsl, target, usl, n, mean, sd, sdw)
     df <- data.frame(sources = sourceVector,
-                     values = valueVector)
+                     values = round(valueVector, N.Decimals))
```

Comment:
if you're rounding here to show rounded data in the tables, you should not do the rounding here, and instead use the `format` argument of the `$addColumnInfo` method of jaspTable.

If you're rounding here because you show some tables with ggplot, you can ignore this.